wire-ios/Wire Notification Service Extension/NotificationService.swift

```diff
@@ -101,8 +102,18 @@ final class NotificationService: UNNotificationServiceExtension {
             return nil
         }
 
-        if DeveloperFlag.multibackend.isOn {
-            // Only new extension is supported in multibackend mode.
+        guard let apiVersion = fetchLastKnownAPIVersion(
+            for: request,
+            appContainerURL: appContainerURL
+        ) else {
+            WireLogger.notifications.critical(
+                "api version unknown, can't load service",
+                attributes: .safePublic
+            )
+            return nil
```

Comment:
question: if nil should we fallback to new NotificationServiceExtension as it will fail later when resolving the backend metadata?